io.github.Bitsafe_cli.yaml

```diff
@@ -0,0 +1,32 @@
+id: io.github.Bitsafe_cli
```

Comment:
github based app-id have 4 components. `io.github`, the user or organization, and the application name.

this doesn't meet the requirements.